Source/WebKit/WPEPlatform/wpe/WPESettings.h

```diff
@@ -162,7 +162,48 @@ typedef enum {
  * Default: 1200
  */
 #define WPE_SETTING_CURSOR_BLINK_TIME "/wpe-platform/cursor-blink-time"
-
+/**
+ * WPE_SETTING_TOPLEVEL_WIDTH:
+ *
+ * The width of the toplevel size.
+ *
+ * VariantType: uint32
+ *
+ * Default: 1024
+ */
+#define WPE_SETTING_TOPLEVEL_WIDTH "/wpe-platform/toplevel-size-width"
+/**
+ * WPE_SETTING_TOPLEVEL_HEIGHT:
```

Comment:
I think we could have a single setting: `WPE_SETTINGS_TOPLEVEL_DEFAULT_SIZE` of type "uu"